device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/IotHubConnectionString.java

```diff
@@ -41,12 +41,12 @@
     private static final Charset CONNECTION_STRING_CHARSET = StandardCharsets.UTF_8;
 
     private String hostName = null;
```

Comment:
any idea why it didn't complain about the other fields, like hostName and deviceId?

Reply:
Not sure, but I can certainly make the same change to them as well